language-support/java/bindings-rxjava/BUILD.bazel

```diff
@@ -54,7 +54,7 @@ testDependencies = [
 ]
 
 da_scala_library(
-    name = "bindings-java-tests-lib",
+    name = "bindings-scala-tests-lib",
```

Comment:
I took the liberty of renaming these.

Reply:
These are tests for the `bindings-java` package, so the naming should stay as is.

Reply:
Ok, reverting the rename.